### ISSUE TITLE
issue with esrgan implementation

### ISSUE THREAD

**Issue by vibss2397** [xinntao/ESRGAN#64]
hi, i was making my own implementation of esrgan, except i made 2 changes, the number of **rrdb blocks is 8** and i **haven't included L1 loss in the perceptual loss function**. The data is **cell microscopy data** and after 100 epochs i have been still getting output like the ones attatched in images( the **generated is the 3rd image**, **ground truth is 4th**, **1st and 2nd are bicubic and nearest neighbor**). does my network need more epochs to train or is there something wrong.

![generated_image_105](https://user-images.githubusercontent.com/22891797/61174838-8f20ad80-a5c3-11e9-8ee7-65772462a897.png)
![generated_image_90](https://user-images.githubusercontent.com/22891797/61174853-a8c1f500-a5c3-11e9-8eb8-e13af627e2d6.png)

**Comment by HarshPathakhp** [comment]
Hey,
I am facing something similar too, did you find a solution to your problem?

**Comment by vibss2397** [comment]
hey @HarshPathakhp, I'm sorry it didn't work i had to switch my strategy

**Comment by XiaotianM** [comment]
L1 loss should not deprecated in loss function
